backend/src/main/java/de/rgmcode/backend/Ticket/Ticket.java

```diff
@@ -0,0 +1,46 @@
+package de.rgmcode.backend.Ticket;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.With;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@With
+@AllArgsConstructor
+@Document("Ticket")
+public class Ticket {
+
+    // theoretische muss noch die User ID pro Ticket für die zuordnung gespeichert werden
+
+    private String id;                              // ID
+    private String ticketUUID;                      // Unique UUID
+    private String ticketDate;                      // Datum wann das Ticket erstellt wurde
+    private String ticketTime;                      // Uhrzeit wann das Ticket erstellt wurde
+
+    private String userID;                          // ID des Erstellers
+
+    private String room;                            // in welchem Raum ist das Problem
+    private String unitID;                          // NB / PC / TC o.ä. ID
+
+    private String customerHeadline;                // Kunden-Überschrift
+    private String customerDescription;             // Kunden-Beschreibung
+    private String ticketStatus;                    // noch offen, in bearbeitung, beendet
+
+    private String processingEmployeeLastName;      // bearbeitender Mitarbeiter
+    private String processingEmployeeFirstName;     // bearbeitender Mitarbeiter
+
+    private String publicComment;                   // öffentlicher Kommentar
+    private String internalComment;                 // interner Kommentar
+
+    private String employeeHistoryLastName;         // Mitarbeiter History
+    private String employeeHistoryFirstName;        // Mitarbeiter History
+
+    private String contributingEmployeesLastName;   // mitwirkende Mitarbeiter
+    private String contributingEmployeesFirstName;  // mitwirkende Mitarbeiter
+
+    private String escalationStatus;                // Eskalastions-Status
+
+    private String solutionAnswer;                  // Lösungs-Antwort
```

Comment:
A lot of redudant comments. I just understand by the variable name `private String id;` that this holds the id. No need to comment that ;-) 

![sign](https://i.redd.it/vs35fh1ruef51.jpg)

Reply:
I will change in the future, it was a help for me to keep the overview